### ISSUE TITLE
[IT] Support the 'Term' template

### ISSUE THREAD

**Issue by BoboTiG** [BoboTiG/ebook-reader-dict#842]
- Wiktionary page: https://it.wiktionary.org/wiki/attribuzione

Wikicode:
```
{{Term|statistica|it}}
```

Output:
```
None
```

Expected:
```
<small>(<i>statistica</i>)</small>
```

---

Model link, if any: https://it.wiktionary.org/wiki/Template:Term

**Comment by BoboTiG** [comment]
The same template is called also lowercased: `term`.

**Comment by BoboTiG** [comment]
This one is for you @atti84it :)

**Comment by atti84it** [comment]
This should be already fixed in #853
